File: generated-sources/kotlin-spring/mojang-api/src/main/kotlin/org/openapitools/model/OrderStatisticsRequest.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.OrderStatistic
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * The body of the request to get the order statistics
 * @param metricKeys 
 */
data class OrderStatisticsRequest (

        @get:NotNull 
        @JsonProperty("metricKeys") val metricKeys: List<OrderStatistic>
) {

}

File: generated-sources/scala-gatling/mojang-api/src/gatling/scala/org/openapitools/client/model/OrderStatisticsRequest.scala

package org.openapitools.client.model


case class OrderStatisticsRequest (
    _metricKeys: List[OrderStatistic]
)
object OrderStatisticsRequest {
    def toStringBody(var_metricKeys: Object) =
        s"""
        | {
        | "metricKeys":$var_metricKeys
        | }
        """.stripMargin
}
